index.js

```diff
@@ -46,11 +46,12 @@ async function asanaOperations(
 try {
   const ASANA_PAT = core.getInput('asana-pat'),
     TARGETS = core.getInput('targets'),
-    TRIGGER_PHRASE = core.getInput('trigger-phrase'),
+    TRIGGER_PHRASE = core.getInput('trigger-phrase').replace(/[|\\{}()[\]^$+*?.]/g, '\\$&')
```

Comment:
I'm not sure I understand what's happening here. I don't think we need it.

Reply:
`|\{}()[]^$+*?.` are special characters in the regular expression and have some specific meaning. To detect them as they are, we need to escape them. `*` means optional or many preceding character or group, to detect an `*` we will have to escape it like `\*`. If we will remove this change end-user will be required to pass a valid regular expression string to detect `TRIGGER_PHRASE` correctly.

Reply:
I understand that you're trying to be helpful, but I think you might be actual making things more difficult to the users. If they want to have special characters in their phrase, they can figure out themselves how to escape them. However, by doing what you're proposing you're making it impossible for them to have regular expressions. So why wouldn't we want for them to be able to do that?

Reply:
My initial understanding was `TRIGGER_PHRASE` will be a static string which will not change across different PRs and we don't need a regular expression to detect a static string. Surely we can allow users to add their own regular expression and it will improve flexibility too. It will be great to describe this part well in the readme so users don't need to tinker around it.

Reply:
I think let's do what gene suggests and add this in the readme.